validator/db/sql/tasks.py

```diff
@@ -337,6 +350,47 @@ async def get_tasks(psql_db: PSQLDB, limit: int = 100, offset: int = 0) -> List[
 
 
 async def get_tasks_by_account_id(psql_db: PSQLDB, account_id: UUID, limit: int = 100, offset: int = 0) -> List[Task]:
```

Comment:
Wasn't this endpoint already in?

Reply:
It was, not sure why git is showing this as completely new - it's a fix